helm/superset/values.yaml

```diff
@@ -362,6 +366,12 @@ ph-ee-engine:
     callback_host: "https://mpesa.sandbox.fynarfin.io/"
     retry_count: 3
     api_timeout: 60000
+    paygops:
+      host: http://ph-ee-connector-ams-paygops
+    roster:
+      host: http://ph-ee-connector-ams-roster
```

Comment:
MPesa as a payment schema connector should not have configurations for AMS connectors.